extensions/data-workspace/src/common/constants.ts

```diff
@@ -56,6 +56,7 @@ export const No = localize('dataworkspace.no', "No");
 export const SdkLearnMorePlaceholder = localize('dataworkspace.sdkLearnMorePlaceholder', "Click \"Learn More\" button for more information about SDK-style projects");
 export const Default = localize('dataworkspace.default', "Default");
 export const SelectTargetPlatform = localize('dataworkspace.selectTargetPlatform', "Select Target Platform");
+export const LocalDevInfo = localize('LocalDevInfo', "Click \"Learn more\" button for more information about local development experience");
```

Comment:
might be helpful to also have the project template name also in this message

Reply:
I was thinking to add the target at the end 'local development experience for [target]' (e.g. 'local development experience for Azure SQL Database'